pkg/templates/updateconfig/internal/params/params.go

```diff
@@ -0,0 +1,33 @@
+package params
+
+// Params represents the params accepted by this template.
+type Params struct {
+	// A regular expression the defines the type of update
+	// strategy allowed.
+	// +required
+	StrategyTypeRegex string
+
+	// The maximum value that be set in a RollingUpdate
+	// configuration for the MaxUnavailable.  This can be
+	// an integer or a percent.
+	// +optional
```

Comment:
You can remove `+optional`, that's the default and our code actually doesn't look at this value

Reply:
Done